clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusClusterCachingAgent.groovy

```diff
@@ -280,7 +280,12 @@ class TitusClusterCachingAgent implements CachingAgent, CustomScheduledAgent, On
     Map<String, CacheData> targetGroups = createCache()
     List<ScalingPolicyResult> allScalingPolicies = titusAutoscalingClient ? titusAutoscalingClient.getAllScalingPolicies() : []
     // Ignore policies in a Deleted state (may need to revisit)
-    Map<String, String> allLoadBalancers = titusLoadBalancerClient ? titusLoadBalancerClient.allLoadBalancers : [:]
+    Map<String, String> allLoadBalancers = [:]
+    try{
+      allLoadBalancers = titusLoadBalancerClient ? titusLoadBalancerClient.allLoadBalancers : [:]
+    } catch(Exception e){
+      log.error("Loading load balancers failed for ${account.name}:${region}", e)
```

Comment:
`"Failed to load load balancers for ${account.name}:${region}"` might be more consistent with the `"Failed to cache ..."` log message also in this class.